src/content/blog/ja/ai-driven-development-workflow.md

```
@@ -0,0 +1,68 @@
+
+## 参考リンク
+
+リンクカードのテスト：
+
+https://cor-jp.com/
+
+https://github.com
+
+---
+
+*AI駆動開発の導入をお考えですか？[お問い合わせ](/contact)からご相談ください。*
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)